lambda/app/src/controllers/user.ts

```diff
@@ -93,7 +93,7 @@ export const updateUserRoleMapping = async (
   },
 ) => {
   const integration = await findAllowedIntegrationInfo(sessionUserId, integrationId);
-  if (integration.authType === 'service-account') throw Error('invalid auth type');
+  //if (integration.authType === 'service-account') throw Error('invalid auth type');
```

Comment:
Are we planning to uncomment these later?

Reply:
we no longer need them so removed them